### PR TITLE
Add description to make the base fee a even number

### PR DESCRIPTION
**Pull request by baerae-colin**
This is required to make the formula true.

block.gas_used * base_fee / 2 == sum of tx.gas_used * base_fee / 2 for all txs.

### CODE REVIEW

File: KIPs/kip-71.md [hunk 1]
```diff
@@ -204,12 +204,23 @@ class World(ABC):
...
+		lower_bound_base_fee = LOWER_BOUND_BASE_FEE
+		upper_bound_base_fee = UPPER_BOUND_BASE_FEE
+
+		# if LOWER_BOUND_BASE_FEE is not a even number, make it even by adding 1
+		if lower_bound_base_fee & 0x1:
+			lower_bound_base_fee += 1
+
+		# if UPPER_BOUND_BASE_FEE ix not a even number, make it even by subtracting 1
+		if upper_bound_base_fee & 0x1:
+			upper_bound_base_fee += 1
```
Comment (kjeom) [inline KIPs/kip-71.md]: for upper_bound_base_fee, we can use a subtraction(-1) strategy in order the nextBaseFee not to exceed it.

Comment (baerae-colin) [reply KIPs/kip-71.md]: Oh, yeap, you're right. I will update it.

Comment (baerae-colin) [inline KIPs/kip-71.md]:
```suggestion
			upper_bound_base_fee -= 1
```

Comment (baerae-colin) [reply KIPs/kip-71.md]: @kjeom Updated. Please take a look again. Thanks!

**Review by jimni1222 — APPROVED** [review]

File: KIPs/kip-71.md (hunk 1 above)
Comment (kjeom) [reply KIPs/kip-71.md]: Thanks!

**Review by kjeom — APPROVED** [review]

**Pull request merged**